### PR TITLE
[d3d9] Implement converter for W11V11U10

### PR DESCRIPTION
**Pull request by K0bin**
Used in D3D8 games. (#1498 and Indiana Jones)

### CODE REVIEW

File: src/d3d9/d3d9_format.cpp [hunk 1]
```diff
@@ -168,6 +168,16 @@ namespace dxvk {
...
 
+      case D3D9Format::W11V11U10: return {
+        VK_FORMAT_B10G11R11_UFLOAT_PACK32,
+        VK_FORMAT_UNDEFINED,
+        VK_IMAGE_ASPECT_COLOR_BIT,
+        { VK_COMPONENT_SWIZZLE_R, VK_COMPONENT_SWIZZLE_G,
+          VK_COMPONENT_SWIZZLE_B, VK_COMPONENT_SWIZZLE_ONE },
+        { D3D9ConversionFormat_W11V11U10, 1u,
+        // Convert -> float (can't use B10G11R11 bc this is a snorm type)
+          VK_FORMAT_R16G16B16A16_SFLOAT } };
```
Comment (DadSchoorse) [inline src/d3d9/d3d9_format.cpp:179]: Why not `VK_FORMAT_R16G16B16A16_SNORM`? Same size and it doesn't have the precision issues close to `1.0`.

Comment (K0bin) [reply src/d3d9/d3d9_format.cpp]: Good call.

File: src/d3d9/d3d9_format.cpp [hunk 2]
```diff
@@ -168,6 +168,16 @@ namespace dxvk {
...
           VK_FORMAT_R16G16B16A16_SFLOAT } };
 
+      case D3D9Format::W11V11U10: return {
+        VK_FORMAT_B10G11R11_UFLOAT_PACK32,
+        VK_FORMAT_UNDEFINED,
+        VK_IMAGE_ASPECT_COLOR_BIT,
+        { VK_COMPONENT_SWIZZLE_R, VK_COMPONENT_SWIZZLE_G,
+          VK_COMPONENT_SWIZZLE_B, VK_COMPONENT_SWIZZLE_ONE },
+        { D3D9ConversionFormat_W11V11U10, 1u,
+        // Convert -> float (can't use B10G11R11 bc this is a snorm type)
```
Comment (Joshua-Ashton) [inline src/d3d9/d3d9_format.cpp:178]: Comment is outdated now

**Comment by AlpyneDreams** [comment]
Looks good to me. I will rebase on this in d8vk if it is merged.

**Pull request merged**